src/Renderer/BlockRenderer.php

```diff
@@ -52,9 +52,15 @@ public function render($block): string
 
         return $this->twig->render('@SetonoSyliusCMSPlugin/block.html.twig', [
             'block' => $block,
+            'renderedBlock' => $this->renderBlockContent($block->getContent() ?? ''),
```

Comment:
```suggestion
            'renderedBlock' => $block->getContent() ? $this->renderBlockContent($block->getContent()) : '',
```
Isn't this a bit cleaner?

Reply:
I think the current code is clean enough. Perhaps you're just more used to the other one